pkg/server/singleprocess/service_deploy.go

```diff
@@ -123,6 +124,35 @@ func setDeploymentUrlIfNeeded(d *pb.Deployment) {
 	}
 }
 
+// TODO: test
```

Comment:
Yes please!

Reply:
Ah, looks like there is a test? Maybe this comment can be removed then.

Reply:
There's a test, the comment is outdated 😄